### PR TITLE
Implement workflow based on bbs+ repo

### PR DESCRIPTION
**Pull request by brownoxford**
This is the first step required for the github workflow based on the `martinthomson/i-d-template` actions. Next will be to move the content of the spec into the skeleton draft document.

### CODE REVIEW

**Comment by brownoxford** [comment]
This addresses #1

**Review by mprorock — APPROVED** [review]
approved, but I unfortunately created conflicts for you - will avoid that in the future

File: README.md [hunk 1]
```diff
@@ -1,33 +1,30 @@
-# JSON Encoding for Post Quantum Signatures
+# CRYSTALS-Dilithium JSON Encoding
```
Comment (mprorock) [inline README.md:1]: this should be updated to JSON Encoding for Post Quantum Signatures

File: README.md [hunk 2]
```diff
@@ -1,33 +1,30 @@
...
 
-Specification for use of Post Quantum Cryptography (PQC) for digital signatures with JSON Encodings
+This is the working area for the individual Internet-Draft, "CRYSTALS-Dilithium JSON Encoding".
 
-## Signatures Supported
+* [Editor's Copy](https://brownoxford.github.io/post-quantum-signatures/#go.draft-post-quantum-signatures.html)
```
Comment (mprorock) [inline README.md:5]: this is pointing at `brownoxford` rather than `mesur-io` as are several other links out

**Review by mprorock — CHANGES_REQUESTED** [review]
see notes around making sure links to github go to the mesur-io repo as well as readme title - otherwise i think good

**Review by mprorock — APPROVED** [review]

**Pull request merged**